app/app/travis.env

```diff
@@ -1,3 +1,3 @@
 CACHE_URL=dbcache://my_cache_table
-DATABASE_URL=psql://postgres:postgres@localhost:5432/postgres
+DATABASE_URL=psql://postgres:postgres@localhost/testdb
```

Comment:
we'll need to update `restore_backup.bash` and `psql.bash` etc

Reply:
@owocki This only affects the `travis` environment.